src/Discord.Net.Commands/Info/CommandInfo.cs

```diff
@@ -125,9 +125,26 @@ public Task<ExecuteResult> Execute(CommandContext context, ParseResult parseResu
             if (map == null)
                 map = DependencyMap.Empty;
 
+            object[] args = null;
+
+            try
```

Comment:
It's safe to leave this try-catch around the whole op, instead of splitting it into two sections.